src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_CAPACITY = 10;
+
+    private K[] keyData = (K[]) new Object[STORAGE_CAPACITY];
+    private V[] valueData = (V[]) new Object[STORAGE_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is the default value of int